files/zh-cn/web/api/navigator/presentation/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Navigator：presentation 属性
+slug: Web/API/Navigator/presentation
+l10n:
+  sourceCommit: b0c8b07682c8d2cecc544f60468f3cf6fc20ac99
+---
+
+{{securecontext_header}}{{APIRef("Presentation API")}}
+
+{{DOMxRef("Navigator")}} 接口的 `presentation` 只读属性是 [演示 API](/zh-CN/docs/Web/API/Presentation_API) 的入口点，用于返回对 {{DOMxRef("Presentation")}} 对象的引用。
```

Comment:
https://www.w3.org/zh-hans/news/2016/w3c-invites-implementations-of-presentation-api/

```suggestion
{{DOMxRef("Navigator")}} 接口的 `presentation` 只读属性是[呈现 API](/zh-CN/docs/Web/API/Presentation_API) 的入口点，用于返回对 {{DOMxRef("Presentation")}} 对象的引用。
```